core/src/main/java/com/amplifyframework/auth/AuthException.java

```diff
@@ -60,8 +60,8 @@ public AuthException(
      */
     public static class InvalidStateException extends AuthException {
         private static final long serialVersionUID = 1L;
-        private static final String MESSAGE = "Auth state reached an invalid state.";
-        private static final String RECOVERY_SUGGESTION = "Please reset auth plugin and reattempt the operation.";
+        private static final String MESSAGE = "Auth state is an invalid state, cannot process the request.";
+        private static final String RECOVERY_SUGGESTION = "Please reset auth plugin or reattempt the operation later.";
```

Comment:
nit: `later` does not sound right. How do customer know when later is?

Reply:
@sktimalsina Agree with these comments, but can we fast-follow with a second PR? This PR will resolve signup's which was otherwise making work difficult.